artistic/service/artistic_photo.py

```diff
@@ -48,11 +48,19 @@ def create(self):
             return resp
         action = ArtisticActions(self.action)
         if action == ArtisticActions.PENCIL_SKETCH:
-            image = photo.pencil_sketch(
-                starting_image=self.starting_image,
-                source_name=self.source_name,
+            if not self.outline_image:
+                image = photo.pencil_sketch(
+                    starting_image=self.starting_image,
+                    source_name=self.source_name,
+                )
+                resp.msg = 'Your sketch has been added'
+            else:
+                image = photo.pencil_sketch_outline(
+                    starting_image=self.starting_image,
+                    source_name=self.source_name,
+                    outline_image=self.outline_image,
                 )
-            resp.msg = 'Your sketch has been added'
+                resp.msg = 'Your sketch has been added'
```

Comment:
You could have just one `resp.msg` outside of the `if/else` block and then don't have to do it twice